res/PHP/dotenv-include.php.tmpl

```diff
@@ -1,6 +1,6 @@
 <?php
 call_user_func(function($dotEnvFile) {
     if (!getenv('APP_ENV') && file_exists($dotEnvFile)) {
-        (new \Symfony\Component\Dotenv\Dotenv())->load($dotEnvFile);
+        (new \Symfony\Component\Dotenv\Dotenv())->loadEnv($dotEnvFile);
```

Comment:
Looks like an easy change, but there are multiple things to consider here:

1. This method is only available since SF version 4.2
1. It is a braking change, considering that now `.env.dist` files are loaded, where they were not loaded before.
1. I'm not 100% sure whether there are use cases, when you do **not** want this behavior

My suggestion to resolve all issues would be:

Make the load behavior configurable (in root composer.json), defaulting to the old behavior.
Configuration options could then be `load` (default), `overload`, `loadEnv`

What do you think?